src/mediastreamtrack.cc

```diff
@@ -80,6 +80,16 @@ NAN_GETTER(MediaStreamTrack::GetEnabled) {
   info.GetReturnValue().Set(Nan::New(self->_track->enabled()));
 }
 
+NAN_SETTER(MediaStreamTrack::SetEnabled) {
+  (void) property;
+
+  auto self = AsyncObjectWrapWithLoop<MediaStreamTrack>::Unwrap(info.Holder());
+
+  CONVERT_OR_THROW_AND_RETURN(value, enabled, boolean_t);
```

Comment:
I think you need `bool` rather than `boolean_t`. We've defined the converter for `bool` [here](https://github.com/node-webrtc/node-webrtc/blob/develop/src/converters/v8.h#L66).

Reply:
I've made the change - some builds still failing. I'll have a look into it over the coming days when I get some time.

Reply:
Thanks, @taylorhoward92. I took a look at the failures, and they're all

```
#
# Fatal error in: ../../download/src/pc/rtpreceiver.cc, line 248
# last system error: 0
# Check failed: !stopped_
# Aborted
```

Line 248 is the first line of the private method `webrtc::AudioRtpReceiver::Reconfigure()`, which is called from `webrtc::AudioRtpReceiver::OnChanged()` and `webrtc::AudioRtpReceiver::SetupMediaChannel(uint32_t ssrc)`. I think, although I haven't confirmed, that the `webrtc::AudioRtpReceiver::OnChanged()` is getting called in response to `webrtc::MediaStreamTrack::set_enabled(bool enable)`.

Reply:
@markandrus I'm struggling to find what the issue is that's resulting in a stopped track when changing the enabled value (probably due to my minimal C++ experience). Is there anything that you suspect might be causing it?